### PR TITLE
[doc] Split readme into quick start and detail.

### PR DESCRIPTION
**Pull request by 130s**
<s>**DO NOT MERGE YET**.</s>

Addresses part of https://github.com/ros-industrial/industrial_ci/issues/105

### CODE REVIEW

**Comment by 130s** [comment]
Ready for review

**Comment by 130s** [comment]
I don't usually do this but let me merge without review. Today is a bit unusual day where we need a good news..

File: industrial_ci/source_tests.sh [hunk 1]
```diff
@@ -1,36 +1,22 @@
 #!/bin/bash
 
-# Software License Agreement (BSD License)
-#
 # Copyright (c) 2015, Isaac I. Y. Saito
+# Copyright (c) 2017, Mathias Luedtke
```
Comment (ipa-mdl) [inline industrial_ci/source_tests.sh]: ü ;)

File: README.rst [hunk 2]
```diff
@@ -1,379 +1,81 @@
...
-2) do not specify `$ROSINSTALL_FILENAME`
-
-Have multiple .rosinstall files per ROS-distro
-++++++++++++++++++++++++++++++++++++++++++++++
-
-By adding `.$ROS_DISTRO` suffix to your `$ROSINSTALL_FILENAME` file, you can specify which file to use per your `$ROS_DISTRO`. So the syntax of the file name for this purpose is `$ROSINSTALL_FILENAME.$ROS_DISTRO`.
-For example, let's say you want to test multiple distros (indigo, jade) and you have `.travis.rosinstall` and `.travis.rosinstall.jade` files in your repo. You can define the Travis config as:
-
-::
+* Note that `.ci_config` is the REQUIRED name of the cloned folder; it is hardcoded on `industrial_ci` side so you must adhere to it.
```
Comment (130s) [inline README.rst]: Thanks, a fix is on its way #115

**Pull request merged**